### ISSUE TITLE
pip install does not work

### ISSUE THREAD

**Issue by bilelgasmi** [jolly-good-toolbelt/sphinx_gherkindoc#26]
**Env:**

1. python -V -> 3.5.3
2. pip --version -> pip 20.0.2 from /usr/local/lib/python3.5/dist-packages/pip (python 3.5)

When I run `pip install sphinx-gherkindoc` I get this error:

```
ERROR: Could not find a version that satisfies the requirement sphinx-gherkindoc (from versions: none)
ERROR: No matching distribution found for sphinx-gherkindoc
```

I when I run `pip install https://github.com/jolly-good-toolbelt/sphinx_gherkindoc/archive/master.zip` I get I bigger error:

```
ERROR: Exception:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/cli/base_command.py", line 186, in _main
    status = self.run(options, args)
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/commands/install.py", line 331, in run
    resolver.resolve(requirement_set)
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/legacy_resolve.py", line 177, in resolve
    discovered_reqs.extend(self._resolve_one(requirement_set, req))
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/legacy_resolve.py", line 333, in _resolve_one
    abstract_dist = self._get_abstract_dist_for(req_to_install)
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/legacy_resolve.py", line 282, in _get_abstract_dist_for
    abstract_dist = self.preparer.prepare_linked_requirement(req)
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/operations/prepare.py", line 516, in prepare_linked_requirement
    req, self.req_tracker, self.finder, self.build_isolation,
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/operations/prepare.py", line 95, in _get_prepared_distribution
    abstract_dist.prepare_distribution_metadata(finder, build_isolation)
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/distributions/sdist.py", line 38, in prepare_distribution_metadata
    self._setup_isolation(finder)
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/distributions/sdist.py", line 96, in _setup_isolation
    reqs = backend.get_requires_for_build_wheel()
  File "/usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/wrappers.py", line 152, in get_requires_for_build_wheel
    'config_settings': config_settings
  File "/usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/wrappers.py", line 255, in _call_hook
    raise BackendUnavailable(data.get('traceback', ''))
pip._vendor.pep517.wrappers.BackendUnavailable: Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/_in_process.py", line 63, in _build_backend
    obj = import_module(mod_path)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 673, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/poetry/masonry/__init__.py", line 10, in <module>
    from .builder import Builder
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/poetry/masonry/builder.py", line 1, in <module>
    from .builders.complete import CompleteBuilder
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/poetry/masonry/builders/__init__.py", line 1, in <module>
    from .complete import CompleteBuilder
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/poetry/masonry/builders/complete.py", line 6, in <module>
    from poetry.factory import Factory
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/poetry/factory.py", line 10, in <module>
    from clikit.api.io.io import IO
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/clikit/__init__.py", line 1, in <module>
    from .api.config.application_config import ApplicationConfig
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/clikit/api/config/__init__.py", line 1, in <module>
    from .application_config import ApplicationConfig
  File "/tmp/pip-build-env-bfznlw14/overlay/lib/python3.5/site-packages/clikit/api/config/application_config.py", line 6, in <module>
    from typing import ContextManager
ImportError: cannot import name 'ContextManager'
```

Any ideas?

**Comment by bradsbrown** [comment]
@bilelgasmi Thanks for reaching out! Due to the way `sphinx-gherkindoc` was built/implemented, it is supported in `python>=3.6` only (note in the Meta section of the [pypi page](https://pypi.org/project/sphinx-gherkindoc/) sidebar).

**Comment by bilelgasmi** [comment]
I indeed missed that part of the documentation. sorry about that.

It works now 🎉 .

Thanks for this great plugin btw 🚀
